modules/os-eap7-launch/added/launch/datasource.sh

```diff
@@ -41,7 +41,7 @@ function generate_datasource() {
   local validate="${13}"
   local url="${14}"
 
-  generate_datasource_common "${1}" "${2}" "${3}" "${4}" "${5}" "${6}" "${7}" "${8}" "${9}" "${10}" "${11}" "${12}" "${13}" "${14}"
+  generate_datasource_common "${1}" "${2}" "${3}" "${4}" "${5}" "${6}" "${7}" "${8}" "${9}" "${driver}" "${11}" "${12}" "${13}" "${14}"
```

Comment:
If / Once replacing `driver`, please replace all positional arguments passed to `generate_datasource_common` with their respective / corresponding local names, not just the `driver` one. IOW change it to read like:

```
generate_datasource_common "${pool_name}" ....
```

Reply:
:+1